Makefile

```diff
@@ -262,16 +262,12 @@ format-md: ## Format all markdown and MDX files to wrap at 80 characters
 .PHONY: lint
 lint: ## Run linter (clippy)
 	cargo clippy --all-targets -- -D warnings \
-	  --allow clippy::large_enum_variant \
-	  --allow clippy::result-large-err \
-	  --allow unused
+	  --allow unused_assignments
```

Comment:
Oh no... for the whole thing!?

Reply:
(I actually don't know the difference between `unused_assignments` vs `unused`)

Reply:
Ideally we take it out when the compiler issue in #1893 is addressed ;)